Source/WebCore/rendering/svg/LegacyRenderSVGShape.cpp

```diff
@@ -101,8 +100,8 @@ bool LegacyRenderSVGShape::shapeDependentStrokeContains(const FloatPoint& point,
 
     if (hasNonScalingStroke() && pointCoordinateSpace != LocalCoordinateSpace) {
         AffineTransform nonScalingTransform = nonScalingStrokeTransform();
-        Path* usePath = nonScalingStrokePath(m_path.get(), nonScalingTransform);
-        return usePath->strokeContains(nonScalingTransform.mapPoint(point), [this] (GraphicsContext& context) {
+        Path usePath = nonScalingTransform.mapPath(path());
+        return usePath.strokeContains(nonScalingTransform.mapPoint(point), [this] (GraphicsContext& context) {
```

Comment:
The lambda on the other hand is repeated four times as-is, and that deserves a helper to avoid the duplication.